### DISCUSSION TITLE
SkyAPM.Agent.AspNet1.3.0是否支持http sqlclient、rabbitMQ、redis、MySQL ？

### DISCUSSION THREAD

**Discussion by StevenWangGitHub** [SkyAPM/SkyAPM-dotnet#550] [Q&A]
SkyAPM.Agent.AspNet1.3.0是否支持http sqlclient、rabbitMQ、redis、MySQL ？我部署的监控不到这些组件，如果支持部署时需要注意哪些？

**Comment by liuhaoyang** [comment]
可以自己看看 v1.3.0 tag 分支的代码确定支持的插件
